test/functional/data.spec.js

```diff
@@ -26,6 +29,10 @@ describe('swag-for-dev', function () {
 		});
 	});
 
+	it('valid alphabetical order', function () {
+		expect(data.map(item => item.name.toUpperCase())).to.be.sorted();
```

Comment:
```suggestion
		const left = data.map(({name}) => name.toUpperCase());
		const right = [...left].sort();

		expect(left).to.deep.equal(right);
```

I prefer not to add dependencies for trivial things. Since this dep is only used once, adding a couple of extra lines of logic is preferred over the dep.

Reply:
Ping @jnnck